Jt/backgrounds copy

Turtlelord26 opened the pull request:
Copypasta from rb-backgrounds and Turtlelord26-Tool_Sets due to nasty merge conflicts. I have a high confidence that I got everything but please doublecheck. EDIT: Oh, the hubris.

Issues outstanding as a result of branching work:
- [X] The plasma torch is once again both a weapon and a tool and should probably only be one of those things?
- [x] There are lots of lovely lockpicking items in the Items doc with well-thought-out rules that should probably merge with their counterparts in the tool sets document.
- [x] Also the Forger's kit is the only tool set that still lacks prose.

Turtlelord26 committed:
Create 04_Backgrounds.txt

Borrowed from rb-backgrounds branch to sidestep merge conflicts.

Turtlelord26 committed:
Create 08_Tool_Sets.txt

Borrowed from turtlelord26-Toolsets branch to sidestep merge conflicts

Turtlelord26 committed:
Add Intricate Skill Rules to Basic Rules

Borrowed from tool sets branch

Turtlelord26 committed:
Add Intricate Skills to Skill Rules

Borrowed from Tool Sets Branch

Turtlelord26 committed:
Remove Bolt Cutters from Items

From toolsets branch.
Original toolsets branch predates these nice lockpicking items that also should be absorbed into tool sets in future work.

Turtlelord26 commented:
Blocking #982 and #963 until resolution of this pr.

Turtlelord26 committed:
Correctly locate tool sets doc (#1008)

Slight mistake with where I created the copy

  Changes to be committed:
	renamed:    CharacterCreation/08_Tool_Sets.txt -> Items/08_Tool_sets.txt

bleehu commented:
Is this ready for review?

Turtlelord26 commented:
I requested them

bleehu commented:
Why is it a draft?

bleehu commented on CharacterCreation/06_Skills.txt:
```diff
@@ -90,9 +94,9 @@ are listed below.
-Medicine:	To repair life forms
-Repair:	To repair machines
+Hacking [I]:	To pwn computer systems
+Medicine [I]:	To repair life forms
```
Let's use the term "organics" or some such, rather than "life forms". Machines count as life under some definitions.

rosebrallier replied:
I would say biological life forms; you and James wanted to avoid saying "organic" to leave space to develop non-carbon, non-synthetic life.

Turtlelord26 replied:
The rules are trending in the direction of standardizing on biological and mechanical character categories. Please don't bring up cyborgs

bleehu reviewed (approved):
one nitpick, otherwise shipit!

Turtlelord26 committed:
Terminological Precision: Medicine & Repair

Turtlelord26 committed:
Delete Copy Pasta

rosebrallier commented on 01_01_Basic_Rules.txt:
```diff
@@ -523,6 +523,30 @@ instead of one.
+
+	Tool sets are listed in the 08_Tool_Sets.txt document under Items. Each 
+tool set has an associated suite of skills it will usually apply to and a set of 
+applicable contexts, subject to the GM's discretion, and a weight and noise level 
```
Were we going to cut or keep the references to weight ratings since it's not a feature that's in the scope of this project?

Turtlelord26 replied:
By default I would leave the empty fields in but it is true that we do not need them.

rosebrallier replied:
@bleehu

rosebrallier commented on CharacterCreation/04_Backgrounds.txt:
```diff
@@ -0,0 +1,247 @@
+Origin, Social Class, and Profession. Choose one perk from each of those
+three categories. For example, if you chose "Trioch System", "Caveat Emptor" and 
+"Star Strider" then your character would start with half a set of nanites, two
+Tunra language dialects and 6 skill in Insight at level 1 and a +10 bonus to skill 
```
Need to replace the first "and" with a comma

rosebrallier replied:
Also, "Tuunra"

rosebrallier commented on CharacterCreation/04_Backgrounds.txt:
```diff
@@ -0,0 +1,247 @@
+	While everyone enters the world in their birthday suit, characters are
+generally well-clothed by the time in their lives they enter a Compound X story.
+At Level 1, every character will gain up to three outfits of clothing from their
+background. In general, these should consist of some combination of casual,
```
"In general" can go bye bye.

rosebrallier commented on CharacterCreation/04_Backgrounds.txt:
```diff
@@ -0,0 +1,247 @@
+outlook and life experience. These outfits do not provide any armor, but may offer
+bonuses to some social skill checks at the discretion of the GM. A character may own
+outfits from the list of examples below or any other reasonable set of clothing.
+Outfits can include both footwear and outerwear.
```
Hmmmm okay I just want to edit this paragraph

rosebrallier commented on CharacterCreation/04_Backgrounds.txt:
```diff
@@ -0,0 +1,247 @@
+Outfits can include both footwear and outerwear.
+
+== Tools of the Trade ==
+	Even when travelling light, characters accumulate posessions and tools as they
```
"traveling"

rosebrallier commented on CharacterCreation/04_Backgrounds.txt:
```diff
@@ -0,0 +1,247 @@
++------------------------+------------------------------------------------------------------------------+
+| Tundari society is concentrated primarily on Tundar Prime, but your family set out into the galaxy.   |
+| Whether by choice or by exile, they have built a new home and you grew up surrounded by diverse alien |
+| species, carving a life out of recently-terraformed rock.						                        |
```
funky line-length

rosebrallier commented on CharacterCreation/04_Backgrounds.txt:
```diff
@@ -0,0 +1,247 @@
++----------------------+-----------+-------------------------------------------------------------------------------+
+| When you're shot at, the fear and adrenaline still pulse through you, but then your training kicks in.           |
++----------------------+-----------+-------------------------------------------------------------------------------+
+| Expert Researcher    | Academic  | Gain advantage on investigation checks to find information                    |
```
"Investigation"

rosebrallier commented on CharacterCreation/04_Backgrounds.txt:
```diff
@@ -0,0 +1,247 @@
+| Oh ya, I had a collegue from these parts. Made a killing last time I was here selling "yoshgas." Don't worry     |
+| about it, it doesn't translate all that well.                                                                    |
++----------------------+-----------+-------------------------------------------------------------------------------+
+| Hunter's Bag of Tricks | BountyHunter | Gain license, L2 chest armor plate, and 5 HD zip cuffs.                     |
```
funky formatting and no space between Bounty and Hunter

rosebrallier commented on CharacterCreation/04_Backgrounds.txt:
```diff
@@ -0,0 +1,247 @@
++----------------------+-----------+-------------------------------------------------------------------------------+
+| You have a legal license for bringing in bounties, dead or alive. You've got some other tools too.               |
++----------------------+-----------+-------------------------------------------------------------------------------+
+| Friends in low places | BountyHunter | You gain +6 skill in Knowledge - Underworld.                              |
```
bountyhunter

rosebrallier commented on CharacterCreation/04_Backgrounds.txt:
```diff
@@ -0,0 +1,247 @@
+-Shorts and a tank top
+
+Professional Clothing
+-Khakis and a polo shirt or button-down and sweater
```
"-Khakis, a polo shirt or button-down, and sweater"
Gotta use that sweet, sweet oxford comma here

rosebrallier commented on Items/08_Tool_sets.txt:
```diff
@@ -0,0 +1,125 @@
+-sealing rivets, flash-freezers, etc. Also has a sterilizing unit 
```
Top of the document got chopped off

Turtlelord26 replied:
@.@

Turtlelord26 replied:
Check the re-added first half of the document too please 🐢

rosebrallier commented on Items/08_Tool_sets.txt:
```diff
@@ -0,0 +1,125 @@
+	Includes four pipe-cleaners of varying sizes, valve oil, bore oil,
+key oil, lanolin, polish, extra strings or reeds, soft rags for appilcation,
+3 small hex wrenches, and string cutters. Allows for basic mantinence of 
+the chosen style of instrument. 
```
-Title implies that the instrument is included, prose does not.
-add "rosin" to supplies list

rosebrallier commented on Items/08_Tool_sets.txt:
```diff
@@ -0,0 +1,125 @@
+the chosen style of instrument. 
+
+Relevant Skill(s): Performance, Repair
+Context: Audial art
```
"Audial art" is a phrase that is used exactly nowhere. "Musical" or "Auditory" is much more in line with convention.

rosebrallier commented on Items/08_Tool_sets.txt:
```diff
@@ -0,0 +1,125 @@
+	Includes two small metal spatulas, one serrated and one smooth, 
+three sizes of double-wire end modeling tools, six wooden shapers,
+calipers, and a sponge. Allows for fine detail work and cutting when 
+sculpting a soft clay object.
```
This prose excludes the many, many other sculpting mediums. Should either edit the title to "Potter's Tools" or the prose to include mediums like stone, wood, metal etc.

rosebrallier reviewed (changes requested):
Have a review! (I'll fix a number of these myself, but there are also some questions and some formatting and copy-pasta concerns.)

Turtlelord26 committed:
Half the document tried to escape.

rosebrallier committed:
Update 04_Backgrounds.txt

Minor wording and language edits

Turtlelord26 committed:
Update Musical Instrument Care Supplies

Name now no longer implies containing an instrument too. We can go the other way just as easily.
Audial -> Auditory
Added the rosin

rosebrallier commented on Items/08_Tool_sets.txt:
```diff
@@ -0,0 +1,243 @@
+screwdrivers, and pliers; reels of wire; soldering supplies; breadboards; 
+resistors; capacitors; inductors; rechargeable temporary power supplies; 
+and a miniaturized 3D circuit printer allow the professional electrician 
+to maintain, repair, and construct all manner of electronic gadgetry.
```
That's a lot of semi-colons; it's not in line with the style of the rest of the tool set descriptions

rosebrallier commented on Items/08_Tool_sets.txt:
```diff
@@ -0,0 +1,243 @@
+Electronic Lock Spoofing Kit ($250):
+	Contains a morphological magnetic strip card, password cracker, 
+disassembly and bypass tools for keypads and fingerprint/retina readers, 
+basic circuitboard tools suitable to shorting dead man's circuits, 
```
"suitable for" or "suited to"

rosebrallier commented on Items/08_Tool_sets.txt:
```diff
@@ -0,0 +1,243 @@
+Weight: 
+
+Forger's Implements ($250):
+	[ph]
```
Still missing some prose

Turtlelord26 committed:
Reformat backgrounds tables

Should have defunkified the lines

Turtlelord26 committed:
Rereformatted backgrounds tables

Now accommodates bounty hunter with a space

rosebrallier commented on CharacterCreation/06_Skills.txt:
```diff
@@ -90,9 +94,9 @@ are listed below.
-Repair:	To repair machines
+Hacking [I]:	To pwn computer systems
+Medicine [I]:	To heal biological life forms and biotic machines
+Repair [I]:	To repair mechanical life forms and abiotic machines
```
Given the (very) tight lvl 1 to early-game economy (few classes can afford $250 for a trauma kit; they can barely afford repair and med kits as it is) this effectively nerfs most character's healing rolls by ~20. Repair and Medicine DCs are already quite high even taking the consumable kits into account; we need to compensate unless we're making the game more lethal on purpose.

Turtlelord26 replied:
I have been intending to give out a toolkit at level one from a selection based on class, which would help. At least Soldier and Hunter (in addition to doc/engi) would probably have healing kits in the selection.

But it is a relevant point that, with disadvantage, a character would fail the DC 35 check to use a medikit/repair kit about half the time (with no skill ranks and +0 attr mod), which is probably a feel bad. As technology advances so does soldier-proofing (usually), so I wonder if we really need to have that check?

rosebrallier replied:
@bleehu

bleehu replied:
At the very least, Doc and Engineer need tools at level 1; I'm pretty sure they already have them.

Turtlelord26 replied:
Tools given

Turtlelord26 committed:
De-semicoloned Electrician's kit

Turtlelord26 committed:
Rename: Sculptor -> Potter's tools

Turtlelord26 committed:
Delete empty weight fields

Easy to re-add after a weight system exists.

Turtlelord26 committed:
Typography

Turtlelord26 committed:
Add improvised weapon rules to plasma torch

preparatory for taking the torch out of melee weapons.

Turtlelord26 committed:
Delete plasma torch

Now lives as a tool set in the tool set file

rosebrallier reviewed (approved):
After the Forger's kit has prose and Doctor and Engineer have their tools, ship-it!

Turtlelord26 committed:
Added Starter Toolsets to some classes

Doc and Engi per PR requirements, plus some others that seemed straightforward to me. Very much a draft 1, if that.

Turtlelord26 commented:
Waiting on #1009

Turtlelord26 committed:
Jt backgrounds copy item tools assimilation (#1009)

* Assimilate fishing rod

* Assimilated elec. lockpick, door ram, door press

* Assimilate bump key

* Delete assimilated tools from Items file

Fishing Pole, Door Ram, pneumatic door press, digital lockpick, bump key, phys lockpicks (pure deletion, redundant with existing phys lockpick tool set)

rosebrallier commented on Items/08_Tool_sets.txt:
```diff
@@ -53,6 +69,15 @@ Context: Electronics, Electrical Components, Circuitboards. Repair,
+Electronic Lockpick ($100):
+   	This is a small device hooks into digital locks and replays recent entries
+and factory standard crypto keys until the lock opens. Requires an INT of 4 and
+knowledge(computers) 6 to use.
```
*Knowledge - Computers (sub-skill formatting)

rosebrallier committed:
Forger's Implements

Wrote prose for Forger's Implements and corrected the formatting of Knowledge - Computers

rosebrallier committed:
Added Backgrounds

Creating a Character needed a Backgrounds section

Turtlelord26 commented:
👍

Reveraine reviewed (approved):
I like it! It all looks good and reads well!
Ship it.

bleehu commented on CharacterCreation/04_Backgrounds.txt:
```diff
@@ -0,0 +1,247 @@
+appointments for a corporate executive, a character's past will inform how they 
+act under pressure, react under fire and interact with teammates.
+
+	In addition to influencing your character's decisionmaking and roleplay, 
```
nitpick: the characters aren't roleplaying; they're just living. The player is roleplaying. Reword?

bleehu commented on CharacterCreation/04_Backgrounds.txt:
```diff
@@ -0,0 +1,247 @@
+checks to move in microgragity.
+
+== Clothing ==	
+	While everyone enters the world in their birthday suit, characters are
```
This made me chuckle.

bleehu reviewed (approved):
Shipit!

rosebrallier committed:
Update 04_Backgrounds.txt

Simplified out a nitpick
